moveit_setup_assistant/moveit_setup_framework/src/urdf_config.cpp

```diff
@@ -98,18 +98,15 @@ void URDFConfig::setPackageName()
     urdf_pkg_name_ = "";
     urdf_pkg_relative_path_ = urdf_path_;  // just the absolute path
   }
-  else
+  // Check that ROS can find the package
+  const std::filesystem::path robot_desc_pkg_path = getSharePath(urdf_pkg_name_);
```

Comment:
nit, getSharePath() could return a std::optional

Reply:
I attempted this change but it ended up affecting multiple other / unrelated files that needed to do `value_or`. May attempt in a future PR.